capellambse/diagram/_json_enc.py

```diff
@@ -67,8 +67,8 @@ def __encode_box(o: diagram.Box) -> object:
             "height": _intround(o.size.y),
             "context": sorted(o.context),
         }
-        if o.label is not None and not o.hidelabel:
-            jsonobj["label"] = _encode_label(o.label)
+        if o.labels is not None and not o.hidelabel:
+            jsonobj["label"] = [_encode_label(label) for label in o.labels]
```

Comment:
It's weird that we encode the `labels` field into a `label` JSON property. We should unify that.